Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -63,6 +63,30 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 * `--topology-pools`: (optional) Comma-separated list of topology-constrained rbd pools
 * `--topology-failure-domain-label`: (optional) K8s cluster failure domain label (example: zone, rack, or host) for the topology-pools that match the ceph domain
 * `--topology-failure-domain-values`: (optional) Comma-separated list of the k8s cluster failure domain values corresponding to each of the pools in the `topology-pools` list
+* `--config-file`: Path to the configuration file, Priority: command-line-args > confie.ini values > default values
+
+### Config-file
+
+Use the config file to set the user configuration file, add the flag `--config-file` to set the file path.
+
+Example:
+
+```console
+/config.ini
+
+cephfs-filesystem-name <filesystem-name>
+rbd-data-pool-name <pool_name>
+...
```

Comment:
This doesn't appear to be in the same format as the example ini I see. 

There is no `[Configuration]` header at the top here. Is that necessary?

There are also not equal signs between keys and values. I'm pretty sure ini requires those.

Reply:
THanks for pointing updated